### PR TITLE
Icons Navbar on SectionED

### PR DESCRIPTION
**Pull request by c17alepe**
I have created and adjusted icons to replace the buttons in #7210

I now need feedback on what to improve. I tried Henriks tip about using the fist for ResultEd but it doesnt work as one color. Here is the results and some alternatives. 

Black lines doesnt work on this purple
image

This inverted one looks to thin
image

This one looks okay but the other ones doesnt have out-lines, so it is not perfectly consistant with the other ones
image

I think the last one looks bearable but id much rather have some suggestions what we could switch it with. I also of course want feedback on the other icons and its positioning. 

Past History:
#7296
#7295

Test site
https://c17alepe.webug.his.se:20002/DuggaSys/sectioned.php?courseid=2&coursevers=97732

### CODE REVIEW

**Comment by c17alepe** [comment]
@HGustavs What do you think?

**Comment by a17pongo** [comment]
looks great I think and seems more intuitive.
I would go for suggestions nr.2

File: DuggaSys/sectioned.php [hunk 1]
```diff
@@ -86,24 +82,32 @@
...
-						<a id='filesBTN' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'><input type='button' value='Files' class='submit-button' title='Show files'/></a>
+						<a id='filesBTN' title='Show files' value='Files' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'>
+								<img class="navButt" src='../Shared/icons/rounded_upload_button.svg'>
+							</a>
 						</div>
 					</td>
 					<td class='access menuButton' style='display: inline-block;'>
 						<div class='access menuButton'>
-						<a id='accessBTN' href='' onclick='navigatePage(this.id, "accessed.php");' oncontextmenu='javascript:navigatePage(this.id, "accessed.php");'><input type='button' value='Access' class='submit-button' title='Give students access to the selected version'/></a>
+						<a id='accessBTN' title='Give students access to the selected version' value='Access' href='' onclick='navigatePage(this.id, "accessed.php");' oncontextmenu='javascript:navigatePage(this.id, "accessed.php");'>
```
Comment (c17johha) [inline DuggaSys/sectioned.php]: Indentation police was here, Use 1 TAB

File: DuggaSys/sectioned.php [hunk 2]
```diff
@@ -86,24 +82,32 @@
...
+							<a id='testsBTN' value='Tests' href='' onclick='navigatePage(this.id, "duggaed.php");' oncontextmenu='javascript:navigatePage(this.id, "duggaed.php");'>
+								<img id='testsBTN' class="navButt" src='../Shared/icons/student_files.svg'>
+							</a>
 						</div>
 					</td>
-					<td class='files menuButton' style='display: inline-block;'>
+					<td class='files' style='display: inline-block;'>
 						<div class='files menuButton'>
-						<a id='filesBTN' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'><input type='button' value='Files' class='submit-button' title='Show files'/></a>
+						<a id='filesBTN' title='Show files' value='Files' href='' onclick='navigatePage(this.id, "fileed.php");' oncontextmenu='javascript:navigatePage(this.id, "fileed.php");'>
```
Comment (c17johha) [inline DuggaSys/sectioned.php]: Indentation police was here, Use 1 TAB

File: DuggaSys/sectioned.php [hunk 3]
```diff
@@ -86,24 +82,32 @@
 						</div>
 					</td>
 
-					<td class='results menuButton' style='display: inline-block;'>
+					<td class='results' style='display: inline-block;'>
 						<div class='results menuButton'>
-							<a id='resultsBTN' href='' onclick='navigatePage(this.id, "resulted.php");' oncontextmenu='javascript:navigatePage(this.id, "resulted.php");'><input type='button' value='Results' class='submit-button' title='Edit student results' /></a>
+							<a id='resultsBTN' title='Edit student results' value='Results' href='' onclick='navigatePage(this.id, "resulted.php");' oncontextmenu='javascript:navigatePage(this.id, "resulted.php");'>
```
Comment (c17johha) [inline DuggaSys/sectioned.php]: Indentation police was here, Use 1 TAB for 87-89

**Comment by c17alepe** [comment]
Fixed indentation
Added tooltip
Changed the fist icon
Added more margin between the icons.
Made hamburger menu have border-radius on all sides
Added subtle shadow hover effect to navButt.

The navbutt hover effect will now appear everywhere on the site. So i might have stepped on toes i shouldnt have now. Let me know if this is a bad change

**Comment by c17alepe** [comment]
The searchbar gets a weird shadow effect now. Is there any other place as well?

File: DuggaSys/sectioned.php [hunk 4]
```diff
@@ -44,18 +44,14 @@
 							<select id="courseDropdownTop" class='course-dropdown' onchange='goToVersion(this)' ></select>
 						</div>
 					</td>
-					<td class='editVers' style='display: inline-block;'>
+					<td class='editVers' style='display: inline-block;margin-left:8px;'>
 						<div class='editVers menuButton'>
-								<button type='button' class='submit-button no-radius' style='width:35px;margin-left:0px' title='Edit the selected version' onclick='showEditVersion();'>
-									<img id='versionCog' style='margin-top:6px' src='../Shared/icons/CogwheelWhite.svg'>
-								</button>
+								<img id='versionCog' class="navButt" title='Edit the selected version' onclick=showEditVersion(); src='../Shared/icons/CogwheelWhite.svg'>
```
Comment (c17johha) [inline DuggaSys/sectioned.php]: Indentation appears wrong for me

**Comment by b17nicsv** [comment]
Does every icon have a tooltip now?

**Comment by c17alepe** [comment]
Yes, now they do.

**Comment by c17johha** [comment]
@b17nicsv yes

**Pull request merged**